### PR TITLE
Update link syntax for web/api/window/opener

### PR DESCRIPTION
**Pull request by win93**
### Description

Update link syntax for web/api/window/opener

### Motivation

The markdown link syntax was visible and it was not a link in two places. This change fixes up the syntax such that the markdown syntax is no longer visible and the links are clickable.

### CODE REVIEW

File: files/en-us/web/api/window/opener/index.md [hunk 1]
```diff
@@ -32,10 +32,10 @@ In the following cases, the browser does not populate `window.opener`, but
 leaves it [`null`](/en-US/docs/Web/JavaScript/Reference/Operators/null):
 
 - The opener can be omitted by specifying
-  `[`rel`](/en-US/docs/Web/HTML/Element/a#rel)=noopener` on a link, or passing
+  [`rel`](/en-US/docs/Web/HTML/Element/a#rel)=noopener on a link, or passing
```
Comment (Josh-Cena) [inline files/en-us/web/api/window/opener/index.md:35]:
```suggestion
  [`rel=noopener`](/en-US/docs/Web/HTML/Element/a#rel) on a link, or passing
```

File: files/en-us/web/api/window/opener/index.md [hunk 2]
```diff
@@ -32,10 +32,10 @@ In the following cases, the browser does not populate `window.opener`, but
...
   `noopener` in the {{domxref("Window.open", "windowFeatures")}} parameter.
 - Windows opened because of links with a [`target`](/en-US/docs/Web/HTML/Element/a#target) of `_blank` don't get an `opener`, unless explicitly
-  requested with `[`rel`](/en-US/docs/Web/HTML/Element/a#rel)=opener`.
+  requested with [`rel`](/en-US/docs/Web/HTML/Element/a#rel)=opener.
```
Comment (Josh-Cena) [inline files/en-us/web/api/window/opener/index.md:38]:
```suggestion
  requested with [`rel=opener`](/en-US/docs/Web/HTML/Element/a#rel).
```

File: files/en-us/web/api/window/opener/index.md [hunk 3]
```diff
@@ -32,10 +32,10 @@ In the following cases, the browser does not populate `window.opener`, but
 leaves it [`null`](/en-US/docs/Web/JavaScript/Reference/Operators/null):
 
 - The opener can be omitted by specifying
-  `[`rel`](/en-US/docs/Web/HTML/Element/a#rel)=noopener` on a link, or passing
+  [`rel=noopener`](/en-US/docs/Web/HTML/Element/a#rel) on a link, or passing
```
Comment (wbamberg) [inline files/en-us/web/api/window/opener/index.md:35]:
```suggestion
  [`rel=noopener`](/en-US/docs/Web/HTML/Attributes/rel#noopener) on a link, or passing
```

File: files/en-us/web/api/window/opener/index.md [hunk 4]
```diff
@@ -32,10 +32,10 @@ In the following cases, the browser does not populate `window.opener`, but
...
   `noopener` in the {{domxref("Window.open", "windowFeatures")}} parameter.
 - Windows opened because of links with a [`target`](/en-US/docs/Web/HTML/Element/a#target) of `_blank` don't get an `opener`, unless explicitly
-  requested with `[`rel`](/en-US/docs/Web/HTML/Element/a#rel)=opener`.
+  requested with [`rel=opener`](/en-US/docs/Web/HTML/Element/a#rel).
```
Comment (wbamberg) [inline files/en-us/web/api/window/opener/index.md:38]:
```suggestion
  requested with [`rel=opener`](/en-US/docs/Web/HTML/Attributes/rel#opener).
```

**Review by wbamberg — APPROVED** [review]
👍 thank you @win93 , and sorry to be so slow to review this fix!

**Pull request merged**